parser_library/src/workspaces/processor_group.cpp

```diff
@@ -111,11 +111,16 @@ std::vector<std::pair<std::string, size_t>> processor_group::suggest(std::string
 
 bool processor_group::refresh_needed(const std::vector<utils::resource::resource_location>& urls) const
 {
-    return std::any_of(urls.begin(), urls.end(), [this](const auto& url) {
-        auto candidate = m_refresh_prefix.lower_bound(url.get_uri());
-        if (candidate == m_refresh_prefix.begin())
-            return false;
-        return url.get_uri().starts_with(*std::prev(candidate));
+    return std::any_of(urls.begin(), urls.end(), [this](const auto& res_location) {
```

Comment:
Missing following includes and possibly others as well:
```
#include <algorithm>
#inlcude <string_view>
#include <variant>
```

The same goes for `parser_library/src/workspaces/workspace_configuration.cpp` at least (probably except for `#include <variant>`)